omnibus/package-scripts/agent-deb/postinst

```diff
@@ -182,4 +182,6 @@ else
     echo "No datadog.yaml file detected, not starting the agent"
 fi
 
+${INSTALL_DIR}/embedded/bin/python "${INSTALL_DIR}/python-scripts/postinst.py" "${INSTALL_DIR}"
```

Comment:
Should we check for the python binary before using it? (same for the python-scripts)

Reply:
I don't think so. They're shipped along with the agent.

Reply:
Yes, but we do the same thing for other binaries above in the script:
https://github.com/DataDog/datadog-agent/blob/jose/persisting-python-integrations/omnibus/package-scripts/agent-deb/postinst#L125-L141

Reply:
I don't know if they're optional. In any case I can add the check you suggest.

Reply:
Added check